app/ide-desktop/lib/types/ipc.d.ts

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
should probably prefer putting this in `types.d.ts`

Reply:
Yea, possibly, but the file is more specific to the IPC, and we already have a ton of files with more than 400 lines, which makes them harder to read, understand and refactor due to unstructured dependencies

Reply:
Having small SRP files IMO much better